spring-data-sample/src/main/java/sample/SampleService.java

```diff
@@ -146,9 +146,8 @@ public String getOrdersByCustomerId(int customerId) {
     // Retrieve the order info for the customer ID from the orders table.
     // Return the order info as a JSON format.
     return asJson(
-        orderRepository.findAllByCustomerId(customerId).stream()
+        orderRepository.findAllByCustomerIdOrderByTimestampDesc(customerId).stream()
```

Comment:
This new API pushdowns the order to the underlying storage and this service class doesn't need to sort the entities